test/test-rotateElement.js

```diff
@@ -0,0 +1,902 @@
+    })
+})
+
+describe("element rotation around the axis 5x5 not on diagonal", function () {
+
+    it("element move the top", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            ----X
+            -----
+            -----
+            -----
+            
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -X---
+            -----
+            -----
+            -----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("element move the left", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -X---
+            -----
+            -----
+            -----
+            -----
+            
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            X----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("element move the right", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            X----
+            -----
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+        let expectedResult = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            ---X-
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+    it("the item returns to its original position", function(){
+        let elementTetromino = tetris.getCoordinates(`
+            -----
+            -----
+            -----
+            -----
+            ---X-
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            -----
+            -----
+            --X--
+            -----
+            -----
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum[0], elementTetromino[0])
+		let expectedResult = tetris.getCoordinates(`
+            -----
+            ----X
+            -----
+            -----
+            -----
+        `)
+        assert.equal(expectedResult[0], stepElementTetromino)
+    })
+
+describe("element rotation around the axis 7x7 not on diagonal", function () {
```

Comment:
Your tests for 7x7 is exact copy of 5x5, but with 1 line of padding on the sides - not very useful - you're not covering new use-cases for your function

Reply:
There will be other coordinates

Reply:
yes, but I would rather see some more interesting tests